test/dogma/rules/trailing_whitespace_test.exs

```diff
@@ -29,6 +29,26 @@ defmodule Dogma.Rules.TrailingWhitespaceTest do
     ]
   end
 
+  with "windows line endings are okay" do
```

Comment:
I don't like the naming of this set of tests.

'with windows line endings are okay it should raise errors' nor 'it should raise errors with windows line endings are okay' read correctly.